ex1/ex1.c

```diff
@@ -9,6 +13,17 @@
 int main(void)
 {
     // Your code here
+    int x;
+
+    x = 100;
+
+    fork();
```

Comment:
This isn't quite how fork() is used. Fork returns a process id and depending on if that process id is 0 or (positive) non-zero you'll execute either the parent or child processes. You'll notice that in this code (at least when I ran it) the process ids are the same on lines 22 and 26. Only thing that seems to have happened here is that you've manually changed the variable x.